### PR TITLE
Task requirements yu

### PR DESCRIPTION
**Pull request by yudjee**

### CODE REVIEW

File: src/components/CourseForm/CourseForm.jsx [hunk 1]
```diff
@@ -11,10 +11,12 @@
 // //   ** user should be able to remove author from the course
 // //   ** add validation to the fields
 // //   ** add new course to the 'coursesList' and navigate to the '/courses' page => new course should be in the courses list
+// ** TASK DESCRIPTION ** - https://d17btkcdsmqrmh.cloudfront.net/new-react-fundamentals/docs/module-2/home-task/components#add-new-course
```
Comment (MillerOlesiaJob) [inline src/components/CourseForm/CourseForm.jsx]: I think here and further should be a double comment.

File: src/services.js [hunk 2]
```diff
@@ -18,15 +18,15 @@ export const getCurrentUser = async () => {
...
+export const updateCourseService = async () => {
   // write your code here
 };
 
 export const logout = async () => {
   // write your code here
 };
 
-export const deleteCourse = async () => {
+export const deleteCourseService = async () => {
```
Comment (MillerOlesiaJob) [inline src/services.js:29]: Why did you add `Service` word here and did not add for other services?

Comment (yudjee) [reply src/services.js:29]: because  of deleteCourse action and deleteCourse service names duplication

File: src/components/CourseForm/CourseForm.jsx (hunk 1 above)
Comment (yudjee) [reply src/components/CourseForm/CourseForm.jsx]: fixed all cases

**Review by MillerOlesiaJob — APPROVED** [review]
